docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -91,13 +91,14 @@ supports the following features:
 * :doc:`/sql/insert`
 * :doc:`/sql/delete`, see also :ref:`iceberg-delete`
 * :ref:`sql-schema-table-management`, see also :ref:`iceberg-tables`
-* :ref:`sql-materialized-views-management`
+* :ref:`sql-materialized-views-management`, see also
+  :ref:`iceberg-materialized-views`
 * :ref:`sql-views-management`
 
 .. _iceberg-tables:
 
 Partitioned tables
-------------------
+^^^^^^^^^^^^^^^^^^
```

Comment:
unrelated changes

Reply:
It is not unrelated - it establishes the Materialized views section and all others under the SQL support

Reply:
See screenshot in description .. without this the order of sections and hierarchy makes no sense